test/server/admin/admin_test.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+#
+# This test is semi-automatic. It builds envoy-static with
+# ENVOY_ADMIN_BROWSER_TEST set, then runs the binary in a mode where
+# the binary picks the admin port and writes it to a file. Then
+# the script can poll the admin port for /ready. Once live, it runs
+# Firefox (could be any browser) on the test HTML page, which will
```

Comment:
we might want to use selenium for this (as a follow up)

Reply:
added comment.